modules/ossm-security-cipher.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-security-cipher_{context}"]
+= Configuring cipher suites and ECDH curves
+
+Cipher suites and Elliptic-curve Diffie–Hellman (ECDH curves) can help you secure your service mesh when you are using TLS 1.2 or earlier. You can define a comma separated list of cipher suites using `spec.istio.global.tls.cipherSuites` and ECDH curves using `spec.istio.global.tls.ecdhCurves` in your `ServiceMeshControlPlane` resource. If either of these attributes are empty, then the default values are used. 
+
```

Comment:
Perhaps add a note along the lines of "Please note the cipherSuites setting will have no effect when negotiating TLS 1.3"

Reply:
@neal-timpe just a suggestion